src/qt/coincontroldialog.cpp

```diff
@@ -485,11 +484,11 @@ void CoinControlDialog::updateLabels(CCoinControl& m_coin_control, WalletModel *
             if (!CoinControlDialog::fSubtractFeeFromAmount)
                 nChange -= nPayFee;
 
-            // Never create dust outputs; if we would, just add the dust to the fee.
-            if (nChange > 0 && nChange < MIN_CHANGE)
+            if (nChange > 0)
             {
```

Comment:
nit in commit 762f0540ed009e5597a923ead557e17941d5e351: You can remove the newline after the `if`. (Only if you need to touch this branch again for other reasons).

Reply:
Done